src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import UserInfo from '../UserInfo/index';
+import CommentList from '../CommentList/index';
```

Comment:
Same suggestion with imports here
```suggestion
import UserInfo from '../UserInfo';
import CommentList from '../CommentList';
```